lms/product/plugin/course_copy.py

```diff
@@ -29,6 +29,10 @@ def find_matching_file_in_course(
         try:
             # Get the current (copied) courses files, that will have the side effect of storing files in the DB
             _ = store_new_course_files(new_course_id)
+        except OAuth2TokenError:
+            # If the issue while trying to talk to the API is OAuth related
+            # let that bubble up and be dealt with.
+            raise
```

Comment:
This makes sense to me: we need `OAuth2TokenError` to be handled differently from other errors